src/msw/graphicsd2d.cpp

```diff
@@ -3759,11 +3764,17 @@ class wxD2DDCRenderTargetResourceHolder : public wxD2DRenderTargetResourceHolder
         wxCHECK_RET( status != ERROR, wxS("Error retrieving DC dimensions") );
 
         hr = renderTarget->BindDC(m_hdc, &r);
-        wxCHECK_HRESULT_RET(hr);
-        renderTarget->SetTransform(
-                       D2D1::Matrix3x2F::Translation(-r.left, -r.top));
 
-        m_nativeResource = renderTarget;
+        // BindDC may fail with E_INVALIDARG if the given RECT is too large.
+        // It is better to let the user code handle this condition than
```

Comment:
The trouble is that currently there is no way for the application code to even know that an error occurred... Could we propagate the error from here somehow?